src/test/java/space/obminyashka/items_exchange/service/MailServiceTest.java

```diff
@@ -39,19 +42,27 @@ void setUp() {
         messageSourceProxy.setMSource(mock(MessageSource.class));
     }
 
-    @Test
-    void sendMail_shouldPassTheFlow() throws IOException {
+    @ParameterizedTest
+    @MethodSource("listEmailType")
```

Comment:
That could be done even more easily with EnumSource annotation that will provide all values of the enum as the incoming test parameter